### PR TITLE
Updated the Typo Mistakes ✅

### PR DESCRIPTION
**Pull request by lavakush07**
Signed-off-by: Lavakush biyani <70131581+lavakush07@users.noreply.github.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c41b4a2be2a4b43e5c95bd81762886d169fb4ca9 at: https://6310907f9a6b4d5ac77e1e08--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
Assigned to @meshery/contributors

**Review by hershd23 — APPROVED** [review]
Good spot. LGTM

**Review by Nikhil-Ladha — APPROVED** [review]
Cheeky peaky!
Thank you :)

File: docs/pages/project/community.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ language: en
...
 ---
 
-Meshery is a CNCF prpoject maintained by engineers from Layer5, VMware, Red Hat, Rackspace, Citrix, Layer5, Instabase, Microsoft, OpenGov, Computas AS, Rill Data, Quantex, Lumina Networks, and HashiCorp. Meshery is an an open source project that adheres to the [CNCF's Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Meshery advance the state of the art surrounding service meshes and enable adopters of Cloud Native technologies and techniques to do so with confidence. Our shared commitment to the open source spirit pushes Meshery forward.
+Meshery is a CNCF project maintained by engineers from Layer5, VMware, Red Hat, Rackspace, Citrix, Layer5, Instabase, Microsoft, OpenGov, Computas AS, Rill Data, Quantex, Lumina Networks, and HashiCorp. Meshery is an open source project that adheres to the [CNCF's Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Meshery advance the state of the art surrounding service meshes and enable adopters of Cloud Native technologies and techniques to do so with confidence. Our shared commitment to the open source spirit pushes Meshery forward.
 
 #### Community First
```
Comment (vedant-kakde) [inline docs/pages/project/community.md:14]:
```suggestion
<p>Meshery is one of a number of open source projects originated by the <a href="https://layer5.io">Layer5 community</a>, which represents the largest collection of service mesh projects and their maintainers in the world.</p>
```
I'm unable to suggest these changes for line number 15. Please check.

Comment (Nikhil-Ladha) [reply docs/pages/project/community.md:14]: What is the change here? 🤔

Comment (vedant-kakde) [reply docs/pages/project/community.md:14]: orginated => originated

Comment (Nikhil-Ladha) [reply docs/pages/project/community.md:14]: Aah...ok.
Maybe another PR?

Comment (vedant-kakde) [reply docs/pages/project/community.md:14]: OK, I'll create one.

**Pull request merged**